base/shared/src/main/scala/scalaz/core/io.scala

```diff
@@ -0,0 +1,71 @@
+package scalaz
+package core
+
+import java.lang.Throwable
+
+import com.github.ghik.silencer.silent
+
+import algebra.MonoidClass
+import ct._
+import types.IsCovariantClass
+import zio.{ IO, Fiber }
+
+trait IOModule
```

Comment:
needed?

Reply:
Don't think so, just following conventions. Can remove it.